src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -70,7 +68,6 @@ def initialize_jobs(self):
                 }
             )
             self._running_jobs = dict()
-            self._completed_job_states = dict()
```

Comment:
Consolidated into Job which also caches state. So you'll have to iterate jm._running_jobs and do job.terminal_state

Reply:
It may be useful to just keep an index of the jobs that are completed, without having the job state attached to it -- e.g. 
```
self._completed_job_states = {
   job_id_1: True,
   job_id_2: True,
}
```

Alternatively, you could store the data in the `_running_jobs` dictionary, e.g.
```
self._running_jobs = {
   job_id_1: { "job": <job object>, "completed": False, "refresh_state": 1 },
   job_id_2: { "job": <job obj>, "completed": True, "refresh_state": 0 },
...
}
```
though you lose some of the advantages of having the list of completed jobs immediately available.

Reply:
Keeping it as a `set` would be simpler, IMO. Maybe just `self._completed_jobs = <set of job ids>`?